build.xml

```diff
@@ -23,6 +23,48 @@ xmlns:artifact="antlib:org.apache.maven.artifact.ant"
 xmlns:maven="antlib:org.apache.maven.artifact.ant"
 xmlns:cs="antlib:com.puppycrawl.tools.checkstyle.ant">
 
+    <!-- ====================================================== -->
```

Comment:
@afine 
Because I need
`<property name="clover.version" value="4.2.1" />`
in
`<property name="clover.jar" location="${clover.home}/lib/clover-${clover.version}.jar"/>`
and the dependency versions at their original location were declared later than clover.jar, so clover.version was undefined at this point.

I had 3 choices:
1. move the clover property set behind the dependency version declarations just before the macro definitions, separating it from the other property set declarations,
1. move all the dependency version declarations before the clover property set,
1. or just declare the clover.version separately from the other dependency versions in the clover property set block.

I've chosen option 2 as I thought that was the best out of the three.
Please let me know if there is a better way to solve this.